### PR TITLE
Add definitions files for KLEMA printers

### PR DESCRIPTION
**Pull request by polyntsev**
Hello. I am a representative of the KLEMA LTD company.
Please add configuration files for printers that we produce.
Thank you.
[definitions.zip](https://github.com/Ultimaker/Cura/files/9919193/definitions.zip)
[extruders.zip](https://github.com/Ultimaker/Cura/files/9919195/extruders.zip)

### CODE REVIEW

**Comment by jellespijker** [comment]
Can you sync your fork, with our `main`. We have added a **printer-linter** tool for our printer definitions and profiles. This should help us with processing our PR's. It performs some standard diagnostic checks and apply auto-formatting on the changed files in this PR, but you can also run it locally. See https://github.com/Ultimaker/Cura/tree/main/printer-linter

If it finds issues for which it knows a fix it should post a comment with that suggested fix such that you can easily commit that fix if you're okay with it.

**Comment by polyntsev** [comment]
> Can you sync your fork, with our `main`. We have added a **printer-linter** tool for our printer definitions and profiles. This should help us with processing our PR's. It performs some standard diagnostic checks and apply auto-formatting on the changed files in this PR, but you can also run it locally. See https://github.com/Ultimaker/Cura/tree/main/printer-linter
> 
> If it finds issues for which it knows a fix it should post a comment with that suggested fix such that you can easily commit that fix if you're okay with it.

Done, is that all? [unit-test] tells me "NameError: name 'actual_version' is not defined"
I don't quite understand what it wants from me :)

**Comment by jellespijker** [comment]
Sorry that you have to be our guinea pig, something in the auto-formatting workflow seems to be faulty. It added a commit in which it formatted to much. Can you revert commit https://github.com/Ultimaker/Cura/pull/13710/commits/04ef14dd6fbd1e62129274bad02fc2dd6a487b56

**Comment by jellespijker** [comment]
> [unit-test] tells me "NameError: name 'actual_version' is not defined"
I don't quite understand what it wants from me :)

Since you're a first-time contributor one of the devs should give the Unit-Test workflow an okay. But for some reason that option isn't show in the GH UI  atm.

I'm investigating

**Comment by polyntsev** [comment]
Sorry, something went wrong with me :)

**Comment by jellespijker** [comment]
No worries, something went wrong with me as well ;-) 

But together we will figure it out

File: resources/definitions/klema_250_twin.def.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        "gantry_height": { "default_value": 0 },
+        "machine_extruder_count": { "value": 2 }, 
+        
+		"machine_heated_bed": { "default_value": true },
+        
+		
+        	
+        "machine_start_gcode": 
+        {
+            "default_value": "M140 S{material_bed_temperature_layer_0} ;\nM190 S{material_bed_temperature_layer_0} ;\nM501 ;\nM104 S150 ;\nG28 ;\nG90 ;\nG0 X0 Y0 F6000 ;\nM109 S{material_print_temperature_layer_0} ;\nG0 Z0.3 F300 ;\nG92 E0 ;\nG1 X70 E10 F500 ;\nG92 E0 ;\nG1 E-1 F500 ;\nG92 E0 ;\nG1 Z1 ;\nG1 X100 F6000 ;\nG1 E-1 F500 ;\nG92 E0"
```
[the remark this answers is not in the document]
Comment (jellespijker) [reply resources/definitions/klema_250_twin.def.json]: Don't commit this suggested fix.

Seems to be a bug in the linter tool, I will look at it tomorrow morning

**Comment by jellespijker** [comment]
You're almost there, The assertion failures mentioned here: https://github.com/Ultimaker/Cura/actions/runs/3594082516/jobs/6057828065#step:13:781 

Should provided you with enough information to finish up this PR

**Comment by polyntsev** [comment]
Your Skynet hates me :)
For every fix it sends a bunch of spam to my mail and git notifications. Google will put it on the spam list soon :)
The errors that Uni Test finds look like:
`NameError: name 'actual_version' is not defined
Error: Process completed with exit code 1.`
`[ERROR] Secret `SLACK_WEBHOOK` is missing. Please add it to this action for proper execution.`

This makes it difficult to find real bugs...

**Review by jellespijker — APPROVED** [review]
LGTM

Thank you for your contribution, the printers should be available in Cura 5.3

**Pull request merged**